CoWorkMobileApp/Models/User.swift

```diff
@@ -9,7 +9,7 @@ import UIKit
 import RealmSwift
 
 class User: Object {
-    @Persisted var id: String = ""
+    @Persisted(primaryKey: true) var id: String = ""
```

Comment:
I'm wondering if you could just pass in the id as a parameter.  So you don't have to do `= ""`  I think this is good for now though.